docs/reference/recognition.md

```diff
@@ -2,12 +2,13 @@
 
 ```
 Recognition {
-  SFFloat  maxRange       100     # [0, inf)
-  SFInt32  maxObjects     -1      # {-1, [0, inf)}
-  SFBool   occlusion      TRUE    # {TRUE, FALSE}
-  SFColor  frameColor     1 0 0   # any color
-  SFInt32  frameThickness 1       # [0, inf)
-  SFBool   segmentation   FALSE   # {TRUE, FALSE}
+  SFFloat  maxRange          100     # [0, inf)
+  SFInt32  maxObjects        -1      # {-1, [0, inf)}
+  SFBool   occlusion         TRUE    # {TRUE, FALSE}
+  SFInt32  occlusionAccuracy 1       # {1, 2}
```

Comment:
I am wondering if we shouldn't also allow a value of 0 for occlusionAccuracy which would allow us to deprecate the occlusion field.

Reply:
Yes, probably it will be simpler to have a single filed.
Ideally, then it would be better to call is simply `occlusion` but it might cause issues with backwards compatibility.

Reply:
I agree it would be better. But since this aims at develop, we can afford to break backwards compatibility. I believe very few Recognition nodes had a non-default value for the occlusion field. So, it shouldn't break many simulations.

Reply:
Ok, so I will apply these changes.